src/components/Homepage/HomePageModal/LoginModal.js

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import HomePage from '../HomePage';
+import Backdrop from '../../Backdrop/Backdrop';
+import Modal from "./Modal";
+import { Link } from "react-router-dom";
+import history from "../../../history";
+import "./Modal.css";
+
+import Google from "../../images/google.3.jpg";
+import Eclispse from '../../images/Ellipse 16.png'
+import Line from '../../images/Line 20.png'
+
+const LoginModal = () => {
+  const closeBtn = () => {
```

Comment:
This can be defined outside this component. 

This is not wrong,  it's just not a common practice to define component inside other components.

I can see 4 components define inside this one functional component.

either move them to a separate file or move them outside this component